.changeset/ten-carpets-think.md

```diff
@@ -1,7 +1,5 @@
 ---
-"@cube-creator/core-api": patch
```

Comment:
Doesn't that mean that it won't create a new version for the API?

Reply:
From this PR alone yes, but the APIs got their own changesets already